docs/timelines.md

```diff
@@ -0,0 +1,133 @@
+Flash               | Flash a playback
+Timed Flash         | Flash a playback with programmed fade times
+Timed Flash and Go  | Timed flash with Go, for a cue list
+Swop                | Swop a playback (cancels all other playbacks while active)
+Preload             | Preload the LTP values of a playback over a number of seconds
+Marker              | Mark a point in the timecode for information
+Wait for go         | Pause the timeline until you press Go to resume (only on internal timecode)
+
+- **Wait for go** triggers are useful when you have a preset sequence followed by an unknown delay, for
+example a win/walk-up sequence for an awards ceremony when you don't know how long the winner
+will take to reach the stage. Shapes will continue to run while the timeline is paused.
+
+Tracks
+------------------
+
+The timeline is split into **Tracks** which can be used to organise triggers into different groups or sections. Each different trigger in the timeline gets its own row in the track. 
+- You can mute tracks to prevent them playing back using the \{Power\} button.
+- You can lock tracks to prevent accidental changes to sections you've finished using the \{Lock\} button.
+- Tracks can be collapsed so they take up less room on the screen, using the arrow button to the left of the track name.
+
+Selecting timecode source
+------------------
+
+You can select one of the four available timecode sources to control your timeline from the [Timeline Options](./timelines/timeline-options.md#timecode-tab).
+
+To set up and enable each of the timecode sources as Internal, MIDI, Clock or SMPTE use the \[Timecode\] softkey at the top level main menu.
+
+Local timecode controls
+------------------
+
+The \{Link\} button in the bottom left of the Timeline View window links or unlinks the timeline with the timecode source. Especially with external timecode, it's useful when recording and editing to be able to control the timecode yourself rather than having to continually ask the video guys to rewind the source. So when this button is set to unlinked, you can control the timeline playback using the buttons in the top left corner of the Timeline View window as you would with an audio player. If you are using Internal Timecode as a timecode source then these buttons will also operate the Internal Timecode source while linked.
+
+![Timeline transport controls](/docs/images/Timeline-Timecode-Transport.png)
+
+- \{ |< \} sets the live timecode back to zero.
+- \{ |> \} starts playing from the start timecode.
+- \{ > \} and \{ || \} are play / pause controls.
+- \{ Stop \} stops playing and sets the live timecode back to zero.
+- \{ Record \} enters live record mode but doesn't start timecode playback.
+
+When you are linked to a timecode (except Internal Timecode), the first four buttons are replaced by the timecode source name and only the Record button is shown.
```

Comment:
I think the transport controls are also there for winamp. This will actually play/pause winamp too.